docs/component-guides/profiles.md

```diff
@@ -135,7 +138,7 @@ The following steps should be run after deploying Kubeflow via the [provided dep
     kubectl apply -f profile_iam.yaml
     ```
 
-1. The configuration can be verified by creating and running a notebook in the following [steps](notebooks.md#try-it-out).
+10. Verify your configuration by [creating and running](/docs/component-guides/notebooks-guide/#try-it-out) a [Kubeflow Notebook](https://www.kubeflow.org/docs/components/notebooks/quickstart-guide/).
```

Comment:
Maybe lets move this to a separate heading called using xyz with notebooks since we will add more in future and this step is not part of configuration.